config/locales/en/sorting.yml

```diff
@@ -0,0 +1,13 @@
+---
+en:
+  sorting:
+    oldest: oldest
+    newest: newest
+    a: A.
+    z: Zed
```

Comment:
Just curious - why is a `A.` but z not `Z.`?

Reply:
Because a single A will be read out as "ah" (it is a word in itself), so we put the dot there to force it to be said in a correct way.

Reply:
Clever!